kubetest/extract_test.go

```diff
@@ -83,3 +83,58 @@ func TestGetKube(t *testing.T) {
 		}
 	}
 }
+
+func TestExtractStrategies(t *testing.T) {
+	cases := []struct {
+		option        string
+		expectURL     string
+		expectVersion string
```

Comment:
Yeah, like the above comment, this happens to be true for the bazel tests I added, but as soon as we add tests for gcs/ci/etc. they will have different `expectURL`s and `expectVersion`s. I can add in a few of those tests just to justify these fields (and demonstrate the expected behavior).